metagraph/src/graph/alignment/dbg_aligner.hpp

```diff
@@ -94,7 +94,8 @@ class ISeedAndExtendAligner : public IDBGAligner {
                     const ISeeder &seeder,
                     IExtender &extender,
                     const std::function<void(Alignment&&)> &callback,
-                    const std::function<score_t(const Alignment&)> &get_min_path_score) const;
+                    const std::function<score_t(const Alignment&)> &get_min_path_score,
+                    bool force_fixed_seed) const;
```

Comment:
what is `force_fixed_seed`?

Reply:
I'll write a comment about this. If it's true, then it forces the extender to use the full length of the seed in alignment (i.e., the resulting alignment has the seed as a prefix). Otherwise, the extender just uses the seed as a starting node in the graph and can take whichever path it chooses.

Reply:
It's done this way because the seeder does not take coordinates or labels into account, so parts of the input seed may have to be ignored.

Reply:
However, for the backwards traversal part of alignment, we know that the input seed (a forwards alignment) is coordinate consistent, so we can tell the extender to use the full seed.